app/publish/index.js

```diff
@@ -90,10 +91,11 @@ module.exports = async function({ testing, shouldGitTag, latestBranch, tagName,
 
     testing || await action();
     const attachments = [];
+    const finalPrefixGitTag = prefixGitTag || publishConfig['tag-version-prefix'] || '';
```

Comment:
```suggestion
    const gitTag = `${gitTagPrefix || publishConfig['tag-version-prefix'] || ''}${version};
```

Then, pass `gitTag` to `git.tag`